src/NuGet.Core/NuGet.Packaging/Signing/Archive/SignedPackageArchiveIOUtility.cs

```diff
@@ -3,24 +3,36 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Globalization;
 using System.IO;
 using System.Linq;
 using System.Security.Cryptography;
 using System.Text;
+using NuGet.Common;
 
 namespace NuGet.Packaging.Signing
 {
     public static class SignedPackageArchiveIOUtility
     {
+        // Central Directory file header size excluding signature, file name, extra field and file comment
+        internal const uint CentralDirectoryFileHeaderSize = 46;
```

Comment:
The name of this variable seems to be misleading, consider renaming it to describe that it excludes signature or update the value and whenever it is used appropriately.